docs/reference/mapping/types/flattened.asciidoc

```diff
@@ -121,6 +121,12 @@ lexicographically.
 Flattened object fields currently cannot be stored. It is not possible to
 specify the <<mapping-store, `store`>> parameter in the mapping.
 
+[NOTE] 
+====
+* The combined length of the key, value, and separator must not exceed the maximum Lucene term length.
```

Comment:
I'm not familiar enough with ES or Lucene internals to know what's reasonable here, but to make the limitation a bit more meaningful I think we should indicate what that maximum term length is. Otherwise an ES client cannot know how to sanitise the input without digging into code.

Is there anything we can link to? Or could we specify the value here?